Documentation/contributing/docs/index.rst

```diff
@@ -9,7 +9,8 @@
 Documentation
 -------------
 
-This section describes the style and testing methods of Cilium documentation.
+This section describes the style and testing methods of Cilium documentation. Before contributing, please review the documentation style guide below.
+See :ref:`step 1-4 (Clone and provision environment section) <howto_contribute>` to see how to fork and clone the repository.
```

Comment:
I wouldn't mention `step 1-4`, because if we ever change these steps in contributing_guide.rst we'll never remember to update this entry. If you want to point to this particular section, you should instead add a target anchor in contributing_guide.rst above that section, and use that instead of `<howto_contribute>`.